src/SFML/Window/Win32/InputImpl.hpp

```diff
@@ -176,6 +177,10 @@ class InputImpl
     ///
     ////////////////////////////////////////////////////////////
     static Vector2i getTouchPosition(unsigned int finger, const Window& relativeTo);
+
+    private:
+
+    static WORD sfScanToWin(Keyboard::Scancode code);
```

Comment:
The sf in `sfScanToWin` seems kind of odd, i.e. making the namespace a prefix. I'd prefer a different name.